### PR TITLE
Pia 2955 helpscreen migration

### PR DESCRIPTION
**Pull request by zladzeyka**
- Add migration for Onboarding screens
 - Add migration for Help screens

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md [hunk 1]
```diff
@@ -1,8 +1,178 @@
...
+
+Maintaining the Component API along with the simpler Screen API required an increasing amount of effort as we added new
+features. We decided therefore to unify both APIs and introduce the ability to inject fully custom UI elements.
+
+The major benefit of the Component API was the ability to use a custom navigation bar. Via
+`GiniConfiguration.shared.customNavigationController` that is still possible with the new public API.
+
+The following steps will help you migrate to the new public API:
+
+* Configure the SDK the same way as before by using `GiniConfiguration`.
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:24]: [Suggestion] Maybe we could mention that this is a singleton now, if they were used to create a new instance of it before?

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md [hunk 2]
```diff
@@ -1,8 +1,178 @@
...
+
+## Text
+
+ Text customization is done via overriding of string resources.
+
+ You can find all the string resources in [Localizable.strings](https://github.com/gini/gini-mobile-ios/blob/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/de.lproj/Localizable.strings).
+
+# UI Elements
+
+Certain elements of the UI can now be fully customized via UI injection. It utilizes view adapter interfaces which you
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:101]: [Suggestion] As we have some of them implemented in the example app, shouldn't we mention it that they can look into that?

**Review by vizaknaidavid — APPROVED** [review]
Looks good, left 2 small suggestions, but consider it accepted even without them. 😄 🥳

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md (hunk 1 above)
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:24]: Good point. 👍🏻 I'll add it.

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md (hunk 2 above)
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md:101]: Yep, very nice. I'll link it. 😊 Thank you.

**Pull request merged**